worlds/landstalker/test/TestUnique.py

```diff
@@ -0,0 +1,25 @@
+from typing import Dict
+
+from . import LandstalkerTestBase
+from worlds.rogue_legacy.Items import RLItemData, item_table
+from worlds.rogue_legacy.Locations import RLLocationData, location_table
```

Comment:
Any particular reason why Rogue Legacy item and location table is imported here? lol

Reply:
Woops, my bad, I used it as a base when I first started working on it.
I must have forgotten to remove those imports, I'll fix that later today 😅

Reply:
The test itself wasn't really relevant for this game, so I just got rid of it instead of fixing the imports.